sw/airborne/modules/e_identification_fr/README.md

```diff
@@ -0,0 +1,6 @@
+E-Identification-FR
+===================
+
+French law required that every drone broacast its position over wifi beacon frames.
+Use this module with an ESP8266 with and code : https://github.com/enacuavlab/FrDroneID/tree/master/airborne_e_id
```

Comment:
Maybe add a copy at the paparazzi organisation?

Reply:
Ok, the best may be to fork it ?